static/templates/message_body.hbs

```diff
@@ -19,7 +19,7 @@
 
     <span class="alert-msg pull-right"></span>
 
-    <span class="message_time{{#if msg.locally_echoed}} notvisible{{/if}}{{#if status_message}} status-time{{/if}}">
+    <span class="message_time tippy-zulip-tooltip{{#if msg.locally_echoed}} notvisible{{/if}}{{#if status_message}} status-time{{/if}}" data-tippy-placement="top">
```

Comment:
I think it would be better to add all 3 properties in the `timerender.js` code, since then the implementation of this tooltip is concentrated in one place.

Reply:
That makes sense to me. Changed.

Reply:
oops, sorry, I didn't push this yesterday. (I probably forgot the `-f` option and assumed I pushed it :grimacing: )